corehq/apps/reports/analytics/esaccessors.py

```diff
@@ -560,6 +561,15 @@ def get_form_ids_having_multimedia(domain, app_id, xmlns, startdate, enddate):
     return form_ids
 
 
+def scroll_case_names(domain, case_ids):
+    query = (CaseES()
+            .domain(domain)
+            .case_ids(case_ids)
+            .fields(['name', '_id'])
```

Comment:
`fields` is deprecated https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/es/es_query.py#L304-L311